### DISCUSSION TITLE
Is this the way to go for privacy?

### DISCUSSION THREAD

**Discussion by ThisIsStupidGitHub** [OrcaSlicer/OrcaSlicer#6734] [General]
I've been looking at (finally) getting my own 3d printer. I've had many suggestions pointing me at BambuLab printers, but the fact that its software is online is a huge turn off for me. Someone suggested Orca would be an alternative for that so both the computer and the printer are completely offline without losing (hardly) any features of a Bambu printer. 

Is this true?

**Comment by LJKraus** [comment]
The Bambu Studio slicer and OrcaSlicer both work exactly the same with or without internet access for the printer.

Firmware updates just recently became possible with a file downloaded to the SD card instead of direct via internet.

The only features that require internet access are the Handy phone app (for remote monitoring) and printing direct from the Makerworld web site.
